ApplyToBecomeInternal/ApplyToBecome.Data/Models/AdvisoryBoardDecision/AdvisoryBoardDecision.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ApplyToBecome.Data.Models.AdvisoryBoardDecision
+{
+	public class AdvisoryBoardDecision
+	{
+		public int ConversionProjectId { get; set; }		
+		public AdvisoryBoardDecisions Decision { get; set; }
+		public bool? ApprovedConditionsSet { get; set; }
+		public string ApprovedConditionsDetails { get; set; }
+		public List<AdvisoryBoardDeclinedReasons> DeclinedReasons { get; set; }
```

Comment:
I assume that the Declined and Deferred decisions are mutually exclusive. Do we need to record both?

Reply:
Its a good point. I've copied this model from the academisation API because thats the way it stores the data. @MartinWheelerMT any reason why you decided to record this info in separate columns.

Reply:
I have thought about the same myself and in hindsight we could have shared some of the fields in the model, when I first designed the back end model it wasn't immediately clear.  For instance the *OtherReason fields could have been a shared field and maybe the list of reasons on the backend as they are stored as string in the db, but I would still want the strong typing into the List.